### PR TITLE
[ADD] Upload files to s3 using boto3 and python

### PR DESCRIPTION
**Pull request by jramnai**
Added python script which will upload files to aws s3 using python and boto3. We can set access type of the file using this file whether we want it to be public or private on s3.
In contribution to #52

### CODE REVIEW

File: Upload_files_to_s3/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Upload files & folders from your machine to Amazon S3
```
Comment (hastagAB) [inline Upload_files_to_s3/README.md]: Please add `usage` , `requirements` section.
See other scripts for reference.

Comment (jramnai) [reply Upload_files_to_s3/README.md]: added

**Comment by hastagAB** [comment]
@jramnai  Please squash your commits so that I can merge it.

**Comment by jramnai** [comment]
@hastagAB, squashed commits.

**Comment by hastagAB** [comment]
Thanks @jramnai ! 
Keep contributing, All the Best! 👍

**Pull request merged**